### DISCUSSION TITLE
Community Meeting 2022-10-05

### DISCUSSION THREAD

**Discussion by cobya** [microsoft/component-detection#260] [Announcements]
📆 Wednesday 5th October 2022
🕙 10AM Pacific Time ([Find in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221005T17))
☎️ [Teams meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmVmMjcxMmEtMmYwYi00MGNiLTg4MDgtODNiMDY5ZjNkYjY3%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22bb41e436-5c77-44e2-b783-1ad2b9f7aa7f%22%7d)

## Agenda

- Upcoming breaking changes (@RushabhBhansali)

## Attendees

- @JamieMagee (Microsoft)
- @RushabhBhansali (Microsoft)
- @amitla1 (Microsoft)
- @cobya (Microsoft)

## Recording

https://user-images.githubusercontent.com/1358764/194121346-0500b6b3-ae29-464f-91b2-d694dbe01f51.mp4

**Comment by JamieMagee** [comment]
- [Issues opened since last meeting](https://github.com/microsoft/component-detection/issues?q=is%3Aissue+created%3A%3E2022-09-07)
- [PRs opened since last meeting](https://github.com/microsoft/component-detection/pulls?q=is%3Apr+created%3A%3E2022-09-07)
